### DISCUSSION TITLE
Add package info to CLI 'craft plugin/list'

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#10667] [Ideas] [answered]
I love the plugin/list command available from the CLI, but do you think it might be possible to add a column for the package name? That is, like "developer/package-name" ...

The plugin/list command makes it dead easy to find the handle for the plugin we want to then disable or uninstall, but we still have to hunt around somewhere else to find the package name and do a composer remove. If the package name were listed in plugin/list it would be much quicker and easier.

I did find out recently I could achieve what I wanted with "grep handle composer.json", but for a long, long time before that I simply didn't think of that kind of thing, and I'm sure others might also be in the same position.

Anyway, no problem if it's not really worth the effort, or is not possible, or if you think it's not something that many people would use much, but (for me, at least) it would be a real workflow improvement! If so, just close the ticket ;-)

Cheers, Andrew

**Comment by brandonkelly** [comment, marked as answer]
Just released Craft 3.7.36 with this change!

**Reply by AndrewMac** [reply]
Wow, thanks so much for the quick response. I forgot to mention there's another even better use case for this, too, which is when I want to browse the package code and/or issues pages. With this change I can quickly copy the package path and then a second later I am over in Github ... I have a macro that runs open https://github.com/{clip}, so it honestly takes only a second or two.

Thanks so much. I really mean it: you guys are great!

**Reply by brandonkelly** [reply]
Hah, glad to help! The GH URIs aren’t necessarily going to be the same as the package names though. Maybe make your macro go with `https://packagist.org/packages/{clip}` instead, and click on the “Issues” link from there.

**Reply by AndrewMac** [reply]
Thanks for the tip! And by the way, I just had the opportunity to do a composer why-not for a plugin that won't update, and all it took was about two seconds flat, so thanks again!

**Reply by AndrewMac** [reply]
Woah, I hadn't visited the page on packagist before, but it has some really useful at-a-glance info, like the composer details (requires, requires-dev, conflicts, etc ... and that immediately gave me a clue about the conflict I'm having, so I'm actually extra-glad for the tip.
